packages/venia-ui/lib/fragments/productDetails.graphql

```diff
@@ -49,6 +49,9 @@ fragment ProductDetails on ProductInterface {
                 store_label
                 use_default_value
                 value_index
+                swatch_data {
+                    value
```

Comment:
Magento already has an optimized thumbnail which I think would be better to use; this also gives you better indicator that its an image instead of color swatch. Apply same to below.

```suggestion
                    ... on ImageSwatchData {
                        thumbnail
                    }
                    value
```